app/views/profiles/intro/_locked.html.haml

```diff
@@ -0,0 +1,17 @@
+.border-1.border-borderColor6.shadow-smZ1.rounded-8.p-32.mb-32
```

Comment:
I have no idea if extracting this bit of markup to a separate view makes sense. Let me know.